app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,49 @@
+class ProjectsController < ApplicationController
+
+  before_action :find_project, only: [:update, :destroyi, :show]
+
+  def create
+    @project = Project.create(project_params)
+
+    if @project.save
+      render json: @project, status: :created
+    else
+      render json: @project.errors, status: :unprocessable_entity
+    end
+  end
+
+  def index
+    @projects = Project.all
+    render json: @projects, status: 200
+  end
+
+  def update
+    if @project.update_attributes(project_params)
+      render json: @project, status: :ok
+    else
+      render json: @project.errors, status: :unprocessable_entity
+    end
+  end
+
+  def show
+    render json: @project
+  end
+
+  def destroy
+    if @project.destroy
+      head 204
+    else
+      render status: 404
```

Comment:
I believe we have a rescue that handles `404` errors, please do check this [link](https://github.com/AgileVentures/WebsiteOneBackendApi/blob/11656b5d3b8ca31ccc54d074b316a16ba619fa00/app/controllers/application_controller.rb#L2)